prism-agent/service/wallet-api/src/main/scala/io/iohk/atala/agent/walletapi/storage/GenericSecretStorage.scala

```diff
@@ -0,0 +1,29 @@
+package io.iohk.atala.agent.walletapi.storage
+
+import io.iohk.atala.shared.models.WalletAccessContext
+import zio.*
+import zio.json.ast.Json
+
+import java.util.UUID
+import scala.util.Try
+
+final case class CredentialDefinitionSecret(json: Json) // to be moved to pollux?
```

Comment:
Yes, it can be replaced by the [CredentialDefinitionPrivate](https://github.com/hyperledger-labs/open-enterprise-agent/blob/2c5bc51fc66b2c62a7c8ba7e25944704c335253f/pollux/lib/anoncreds/src/main/scala/io/iohk/atala/pollux/anoncreds/Models.scala#L133) class that already exists in pollux 👍

Reply:
Thanks for pointing out. As discussed offline, this will be resolved in future PR or future ZKP work.